ciao/cmd/show.go

```diff
@@ -0,0 +1,202 @@
+// Copyright © 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/ciao-project/ciao/ciao/tool"
+	"github.com/spf13/cobra"
+)
+
+var showCmd = &cobra.Command{
+	Use:   "show",
+	Short: "Show information about various ciao objects",
+	Long:  `Show outputs a list and/or details for available commands`,
+}
+
+var eventShowCmd = &cobra.Command{
+	Use:  "event [TENANT ID]",
```

Comment:
Let's come up with a consistent way to document mandatory vs optional positional arguments. man pages seem to use 
[foo] for optional and foo for non optional.

Reply:
+1